fpga/litex/README.md

```diff
@@ -0,0 +1,52 @@
+# uFork LiteX
+
+An ongoing attempt to package the uFork FPGA design as a
+[LiteX](https://github.com/enjoy-digital/litex) component.
+
+The preliminary goal is to:
+- build a LiteX design without a CPU
+- configure it to use [Litescope](https://github.com/enjoy-digital/litescope)
+  (basically an FPGA debugger)
+  [Hint](https://github.com/enjoy-digital/litescope/issues/28)
+- install on a Fomu
+- blink an LED via Litescope
+
+## How To Run
+
+Run `nix-shell` from this directory. It will automatically install some LiteX
+dependencies, pinned to working versions. If necessary, it will then run
+`./update.sh` to install the latest version of LiteX.
+
+LiteX can be updated by running `./update.sh`.
+
+So far, it is possible to build and install a basic LiteX design by running
+
+    litex_repos/litex-boards/litex_boards/targets/lattice_ice40up5k_evn.py \
```

Comment:
I would have expected to use `litex_repos/litex-boards/litex_boards/targets/kosagi_fomu.py`

I think the board you've selected is the [iCE40 UP5K evaluation board](https://www.latticesemi.com/en/Products/DevelopmentBoardsAndKits/iCE40UltraPlusBreakoutBoard) from Lattice.

Reply:
I totally missed that!